cfgmgr/buffer_pool_mellanox.lua

```diff
@@ -43,9 +40,13 @@ local function iterate_all_items(all_items)
         port = string.match(all_items[i], "Ethernet%d+")
         if port ~= nil then
             local range = string.match(all_items[i], "Ethernet%d+:([^%s]+)$")
-            local profile = redis.call('HGET', all_items[i], 'profile')
-            local index = find_profile(profile)
-            if index == 0 then
+            local profile_name = redis.call('HGET', all_items[i], 'profile')
+            if not profile_name then
+                return 1
```

Comment:
as this is not a valid scenario will it be possible to add an error information for better debuggability?

Reply:
We are not able to add log in lua plugins. But the result of lua will be logged by the buffer manager. By doing so, we will have info to debug.